learn-spam.rspamd.script

```diff
@@ -1,15 +1,86 @@
 #!/bin/bash
-RSPAMD_CONTROLLER_PASSWORD_FILE="/etc/dovecot/rspamd-controller-password"
```

Comment:
I wonder if we should then rename the script to learn.rspamd.script and treat both the action based names as symlinks,

Reply:
but where to put that script? not in the sieve pipe directory, because it shouldn't get called directly.

anyway: with a separate file new questions arise: symlink, `source` or `exec` ? :)

anyway, that is imho a question that can be solved some other time.